OpenRA.Game/Game.cs

```diff
@@ -653,10 +656,10 @@ static void LogicTick()
 		{
 			PerformDelayedActions();
 
-			if (OrderManager.Connection.ConnectionState != lastConnectionState)
+			if (connection != null && connection.ConnectionState != lastConnectionState)
```

Comment:
What would be wrong with using `OrderManager.Connection != null && ...` here instead of a static variable?

Reply:
This sounds reasonable to me, except `OrderManager.Connection` can even ever be null so it simplifies even further. What do you think @teinarss?

Reply:
We still need to check that the OrderManager.Connection is a NetworkConnection

Reply:
Would something like this work?
```csharp
if (OrderManager.Connection is NetworkConnection nc && nc.ConnectionState != lastConnectionState)
{
	lastConnectionState = nc.ConnectionState;
	ConnectionStateChanged(OrderManager, null, nc);
}
```